c2corg_ui/templates/waypoint/helpers/view.html

```diff
@@ -423,3 +423,27 @@
     </div>
   % endif
 </%def>
+
+<%def name="get_waypoint_access_field(locale, waypoint_type, open_tab=False)">\
+  % if locale.get('access'):
+    <%
+        if waypoint_type == 'access':
+            attribute_name = 'pt access'
+        elif waypoint_type in ['hut', 'climbing_indoor', 'climbing_outdoor']:
+            attribute_name = 'pedestrian access'
+        else:
+            attribute_name = 'road or pedestrian access'
```

Comment:
> Should we change it here too?

I guess so. At least it would be consistent to use the same approach for both functions. Make your choice :)